src/cascadia/TerminalSettingsEditor/GlobalAppearance.xaml

```diff
@@ -79,7 +77,7 @@
             <!--  Disable Animations  -->
             <!--  NOTE: the UID is "DisablePaneAnimationsReversed" not "DisablePaneAnimations". See GH#9124 for more details.  -->
             <local:SettingContainer x:Uid="Globals_DisableAnimationsReversed">
-                <ToggleSwitch IsOn="{x:Bind State.Globals.DisableAnimations, Mode=TwoWay, Converter={StaticResource InvertedBooleanConverter}}" />
+                <ToggleSwitch IsOn="{x:Bind local:Converters.InvertBoolean(State.Globals.DisableAnimations), BindBack=State.Globals.SetInvertedDisableAnimationsValue, Mode=TwoWay}" />

```

Comment:
This is a stretch, but could we do something like this so that we don't need the settings model changes?
```suggestion
                <ToggleSwitch IsOn="{x:Bind local:Converters.InvertBoolean(State.Globals.DisableAnimations), BindBack=State.Globals.DisableAnimations(local:Converters.InvertBoolean(<SOME KIND OF REFERENCE TO ToggleSwitch.IsOn??> )), Mode=TwoWay}" />
```

Reply:
Unfortunately, this does not work either since BindBack wants a reference to a function taking one argument, anything that would be similar to your idea would not return a function requiring one argument. BindBack does not feel as intuitive or easy to use as I though it would be 😢